.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "examples/plugin/wasi-logging/wasi-logging"]
```

Comment:
I would like to know how many files we need. If we only need the `wit` file, then we can just download the file directly instead of cloning the whole project.

Reply:
Yes, we only need the `wit` file. But I think using a submodule here is a better approach, based on the following reasons.
1. The original wasi-logging wit files may be changed in the future. If we add the wit files directly into our repo, once the original wasi-logging wit files are updated, we need to create PRs to follow the updates.
2. Since it is a submodule, only the people who need to run the example need to clone the wasi-logging repo. That is, it will not affect the `git clone` users.
3. Wasi-logging repo is a relatively small one. It should not take too much efforts for users to clone.

Anyways, if you still prefer adding only the wit files into repo, I’m willing to make the change :-)

Reply:
For 1, We will also need to create PRs to update the submodule. It's the same issue.
For 2, I think it should be a README or something to talk about how to build and run the example instead of adding more dependencies into such a folder. 
For 3, It's not guaranteed that the size will keep small in the future.

However, the component model is in active development, which means the WIT may change repeatedly. I do not prefer adding only the wit files into the repo; instead, a document with a specific toolchain version, a fixed commit hash, and steps to build is better. And we can use this information in the CI workflow to verify it.

Reply:
Thanks for your feedback, I agree for all.

As for the toolchain version
1. The version of `wit_bindgen` have been specified in Cargo.toml file. 
2. The steps to build are documented in README file. 

The only thing we haven't done yet is documenting the wasi-logging commit hash in the README file. 

So now I'm going to make the change as your request, also speciry the wasi-logging commit hash in the README file.

Reply:
Fixed